README.md

```diff
@@ -181,7 +181,15 @@ For efficient searches we recommend to install `pt` ([the platinum searcher][]).
    fork Spacemacs safely use the `develop` branch where you handle the update
    manually.
 
-3. Launch Emacs. Spacemacs will automatically install the packages it requires.
+3. Spacemacs can be used with Emacs working in a terminal, or full GUI mode. In
+order to do this, launch Emacs as you normally would. The first time you start
+emacs after following the installation instructions mentioned hereby, emacs will
+use the new configuration, automatically downloading and installing additional
+support, not built into Emacs. You'll be able to further customize and refine
+the default configuration by editing the .spacemacs file. Follow this guide to
+see how to do it. The following tutorial will deliver a much more in depth way
```

Comment:
Could you link to `this guide` and drop the last sentence or the other way round?